aries-site/src/data/wcag/datatable.json

```diff
@@ -0,0 +1,518 @@
+    "label": "Keyboard (No Exception)",
+    "status": "failed",
+    "notes": "no keyboard support for datatable column resize"
+  },
+  {
+    "rule": "2.1.4",
+    "label": "Character Key Shortcuts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.1",
+    "label": "Timing Adjustable",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.2",
+    "label": "Pause, Stop, Hide",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.3",
+    "label": "No Timing",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.4",
+    "label": "Interruptions",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.5",
+    "label": "Re-authenticating",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.6",
+    "label": "Timeouts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.3.1",
+    "label": "Three Flashes or Below Threshold",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.3.2",
+    "label": "Three Flashes",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.3.3",
+    "label": "Animation from Interactions",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.1",
+    "label": "Bypass Blocks",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.2",
+    "label": "Page Titled",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.3",
+    "label": "Focus Order",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.4",
+    "label": "Link Purpose (In Context)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.5",
+    "label": "Multiple Ways",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.6",
+    "label": "Headings and Labels",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.7",
+    "label": "Focus Visible",
+    "status": "failed",
+    "notes": "groupby expand/collapse does not receive indicator when navigating with keyboard"
+  },
+  {
+    "rule": "2.4.8",
+    "label": "Location",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.9",
+    "label": "Link Purpose (Link Only)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.10",
+    "label": "Section Headings",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.11",
+    "label": "Focus Not Obscured (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.12",
+    "label": "Focus Not Obscured (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.13",
+    "label": "Focus Appearance",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.1",
+    "label": "Pointer Gestures",
+    "status": "not-applicable",
```

Comment:
I think this would be failed, user can't operate the resize column control without path based gestures

Reply:
the resizable? so failing because the user cant move it using touch?

Reply:
Originally I thought it was failing because it requires path based gestures, but just looked back at the WCAG section where it talks about dragging and path based gestures and it says that dragging is not a path based gesture because "movement from start point to end point does not require the user to follow any particular path or direction." https://www.w3.org/WAI/WCAG22/Understanding/pointer-gestures.html.

With that I'm good with leaving this as not-applicable

Reply:
okay changed back to  not-applicable